src/main/java/com/tstepnik/planner/controller/TaskController.java

```diff
@@ -47,7 +53,7 @@ public TaskController(TaskService taskService, UserService userService) {
 
     @PutMapping("/{id}")
     @PreAuthorize("hasRole('USER')")
-    public ResponseEntity<Task> updateTask(@Valid @RequestBody Task task, @PathVariable("id") Long taskId) {
+    public ResponseEntity<Task> updateTask(@Valid @RequestBody Task task, @PathVariable("id") @RequestParam Long taskId) {
```

Comment:
Same here. You should read what each one of them does.

Reply:
I removed requestbody where i need variable int Url.